### PR TITLE
CLI export command

### PR DESCRIPTION
**Pull request by dominikl**
Adds a basic documentation for the CLI `export` command.

### CODE REVIEW

File: omero/users/cli/export.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Export images
+-------------
+
+The |CLI| export command allows you to export data in XML and TIFF formats from
```
Comment (will-moore) [inline omero/users/cli/export.rst]: Maybe OME-TIFF instead of TIFF, since they are quite different.

Comment (will-moore) [reply omero/users/cli/export.rst]: Or ```ome-tiff``` to be consistent with below (not sure what we do elsewhere?)

Comment (hflynn) [reply omero/users/cli/export.rst]: We use OME-TIFF in the format docs so should be consistent with that here please

File: omero/users/cli/export.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Visit :doc:`overview` to get a basic overview of the |CLI|.
+
+Installation
+^^^^^^^^^^^^
+
+Visit :doc:`installation` to install the |CLI|.
+
+Export command
+^^^^^^^^^^^^^^
+
```
Comment (jburel) [inline omero/users/cli/export.rst:19]: Indicate that you can currently export data in XML and TIFF formats only

File: omero/users/cli/export.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+.. program:: omero export
+
+To export an image as ome-tiff :file:`image.tif`, use::
```
Comment (jburel) [inline omero/users/cli/export.rst]: it is not any image it is image with ID 1

File: omero/users/cli/export.rst [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    $ bin/omero export --file image.xml --type XML Image:<id>
+
+Some of the options available to the export command are:
+
+.. option:: --iterate Dataset:<id>
+
+    Iterate over an object and write individual objects to the directory
+    named by --file (EXPERIMENTAL, the only supported object is Dataset:<id>)
+
+        $ bin/omero export --file output-dir  --iterate  Dataset:2
```
Comment (jburel) [inline omero/users/cli/export.rst]: replace ``2`` by ``<id>`` to match the other command

**Comment by jburel** [comment]
Thanks for adding some docs 
looks good 
merging

**Pull request merged**